strr-web/stores/strr.ts

```diff
@@ -219,6 +273,9 @@ export const propertyDetailsSchema = z.object({
 export const formState: CreateAccountFormStateI = reactive({
   primaryContact,
   secondaryContact,
+  isPropertyManagerRole: false,
+  hasPropertyManager: true,
```

Comment:
True by default? The `Property host` is the default option in UI, so this should be false?

Reply:
`hasPropertyManager` is for `yes` or `no` option and `yes` is the default.

Reply:
From the ticket ` "Property Host" is the default to "what is your role" question and "Yes" is the default to "do you have a property manager who manages this rental unit on your behalf"
`

Reply:
Yes, sorry, you are correct, the Property Manager section is shown by default.